service/src/main/java/com/commercetools/pspadapter/payone/transaction/IdempotentTransactionExecutor.java

```diff
@@ -95,22 +105,22 @@ public PaymentWithCartLike executeTransaction(PaymentWithCartLike paymentWithCar
      * @return 0 if no notifications received yet, else the highest sequence number received + 1
      */
     protected int getNextSequenceNumber(final PaymentWithCartLike paymentWithCartLike) {
-        IntUnaryOperator increase = (x) -> x + 1;
         Predicate<String> isInteger = (i) -> i != null && i.matches("-?[0-9]+");
 
         return IntStream.concat(
             getCustomFieldsOfType(paymentWithCartLike, CustomTypeBuilder.PAYONE_INTERACTION_NOTIFICATION)
                 .map(f -> f.getFieldAsString(CustomFieldKeys.SEQUENCE_NUMBER_FIELD))
-                .filter(isInteger::test)
+                .filter(isInteger)
```

Comment:
😄

Reply:
i hate this function at all, but again - legacy code which will be re-factored non in this PR